aws/fsx/module.tf

```diff
@@ -0,0 +1,161 @@
+terraform {
+  required_version = "~> 0.12.0"
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Module Standard Variables
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "name" {
+  type        = string
+  default     = "fsx"
+  description = "The name of the module"
+}
+
+variable terraform_module {
+  type        = string
+  default     = ""

```

Comment:
- [ ] Set FSX terraform_module to `gravicore/terraform-gravicore-modules/aws/fsx`